### DISCUSSION TITLE
What should I expect to see in a networkingMode=mirrored instance?

### DISCUSSION THREAD

**Discussion by byjrack** [microsoft/WSL#10524] [Q&A] [answered]
So been testing 2.0 to see if it is more compatible with our VPN.  Currently the VPN sets the allocated subnetIP to it's virtual adapter breaking layer3 for most traffic.  We have been using wsl-vpnkit as a path around it, but much rather go to an in box config.

For the moment it isn't making a difference, but also not seeing much of a difference in the instance itself for IP use etc.  Is there a method to know that the instance is configured correctly?  Done a wsl --shutdown and restart of distros and no change.

**Comment by blakeduffey** [comment]
Same problem, been looking for solutions for years - currently using wsl-vpnkit.  Wanted to test the experimental dnsTunneling feature but only works on dev versions of Win11 (doesn't help in corporate environment)

**Comment by byjrack** [comment, marked as answer]
Yeah so decided to watch the announcement video and it helped to provide some context around what I should see. https://youtu.be/OB2Rfzy5V0Y?t=371

Basically `ipconfig /all` should match `ip addr` results which is sort of what I expected per the comments.

And then at the tail end it mentions that this is only available for 11 Insiders which explains why I wasn't seeing any differences. 
https://youtu.be/OB2Rfzy5V0Y?t=751
